general/general.md

```diff
@@ -11,3 +11,11 @@ It takes sometime to sync the progress. If it doesnt sync for a long time, ask f
 ### Q3. Why I am facing "Nothing to compile" issue while compiling my contract in Windows Device?
 
 You need to check and Update your `Hardhat` Version to "2.9.6". You can do this by entering this command ` npm install --save-dev "hardhat@^2.9.6" `
+
+### Q4. How can I get the most out of the learn web 3 platform?
+
+lw3 is so much more than a bunch of tutorials. The projects are very easy to follow and if you allow it will just spook feed you the code, but you learn nothing like that. Here are the best ways to make the most out of all the tracks.
```

Comment:
"LW3" instead of "lw3"
"spoon" instead of "spook"

Reply:
done